tapir/translations/locale/de/LC_MESSAGES/django.po

```diff
@@ -1166,6 +1166,11 @@ msgstr "Rote Karte Lebensmittel"
 msgid "First Aid"
 msgstr "Erste Hilfe"
 
+#: shifts/models.py:40
+msgid "Welcome Session"
+msgstr "Willkommensrunde"
```

Comment:
I think it's "Willkommenstreffen", at least it's how it's worded [here](https://www.eventbrite.com/e/supercoop-willkommenstreffen-welcome-session-tickets-169946593837).